rust/src/disassembly.rs

```diff
@@ -210,3 +216,63 @@ impl Drop for DisassemblyTextLine {
         }
     }
 }
+
+pub use binaryninjacore_sys::BNDisassemblyOption as DisassemblyOption;
+
+#[derive(PartialEq, Eq, Hash)]
+pub struct DisassemblySettings {
+    pub(crate) handle: *mut BNDisassemblySettings,
+}
+
+impl DisassemblySettings {
+    #[allow(unused)]
+    pub(crate) unsafe fn from_raw(handle: *mut BNDisassemblySettings) -> Ref<Self> {
+        debug_assert!(!handle.is_null());
+
+        Ref::new(Self { handle })
+    }
```

Comment:
Comment this out for now; when someone eventually needs it, I'd rather not have the tag lying around.... Unless Cargo proactively warns you that you're using an "allow(unused)"?  Either way, I'd like for it to be loud when we start using this.

Reply:
Adressed in 8b5ab443ae0d7e29c2080aca3fe9f897433a7f45